src/Native/HMAC.pack.js

```diff
@@ -0,0 +1 @@
+window.Crypto = require("crypto-js");
```

Comment:
I think I understand the reason for using `window` here. To be able to access the result of webpack's build in `HMAC.src.js` below right?

I thought that this would introduce a dependency on the browser. In node if I try to `require('./src/Native/HMAC)` I get:

```
ReferenceError: window is not defined
    at Object.<anonymous> (/Users/kevin/bc/aws-sdk-elm/src/Native/HMAC.js:6821:1)
    ...
```

However, running `elm test` does succeed, and you added a unit test to exercise `HMAC` indirectly. So I'm not sure